hazelcast/src/main/java/com/hazelcast/internal/partition/operation/MigrationOperation.java

```diff
@@ -123,6 +135,7 @@ private void logMigrationCancelled() {
         getLogger().warning("Migration is cancelled -> " + migrationInfo);
     }
 
+    /** If the migration was successful, set the partition replica versions from the destination replica index onwards */
```

Comment:
`Sets the partition replica versions from the destination replica index onwards, if the migration was successful.`